activerecord/lib/active_record/connection_adapters/postgresql_adapter.rb

```diff
@@ -414,7 +414,7 @@ def default_index_type?(index) # :nodoc:
         def translate_exception(exception, message)
           return exception unless exception.respond_to?(:result)
 
-          case exception.result.try(:error_field, PGresult::PG_DIAG_SQLSTATE)
+          case exception.result.try(:error_field, PG::PG_DIAG_SQLSTATE)
```

Comment:
`PGresult` has been renamed to `PG::Result` [source](https://github.com/ged/ruby-pg/blob/master/History.rdoc#v0130-2012-02-09-michael-granger-gedfaeriemudorg)